core/geometry/src/test/polyface/OffsetMeshContext.test.ts

```diff
@@ -221,7 +221,7 @@ describe("OffsetMeshContext", () => {
     GeometryCoreTestIO.saveGeometry(allGeometry, "OffsetMeshContext", "OffsetDTM");
     expect(ck.getNumErrors()).equals(0);
   });
```

Comment:
for got to remove ".only"
will remove them in the next commit  hopefully along with the fix for these 2 tests!